src/zap/zapimage.cpp

```diff
@@ -1547,7 +1547,7 @@ void ZapImage::OutputTables()
 
 #if defined(FEATURE_PAL)
     // PAL library requires native image sections to align to page bounaries.
-    SetFileAlignment(0x1000);
+    SetFileAlignment(0x10000);
```

Comment:
@jkotas I was suggesting making this and the next change constant across all platforms, but one other option would for it to match the OS_PAGE_SIZE.  This would require native images to be generated with the matching page size.  It depends on the deployment model planned.